components/mdns/mdns.c

```diff
@@ -334,6 +332,9 @@ static mdns_host_item_t *mdns_get_host_item(const char *hostname)
 
 static bool _mdns_can_add_more_services(void)
```

Comment:
If return false, could we print some logs for the user to let them know the configuration `MDNS_MAX_SERVICES` is insufficient to add more services? For now, in the function [mdns_service_add_for_host](https://github.com/espressif/esp-protocols/blob/master/components/mdns/mdns.c#L5904) we will leave an error log for the user `Cannot add more services` and return an error `NO_MEMORY`. The user might see, `Oh, memory might be used up`. But if they print the heap usage debugging log, there might be a lot memory left. It might comfuse the user. Some issues from the user for asking this: https://github.com/espressif/esp-thread-br/issues/115

Reply:
Good point, have updated the error message to:
```cpp
                      "Cannot add more services, please increase CONFIG_MDNS_MAX_SERVICES (%d)", CONFIG_MDNS_MAX_SERVICES);
```